src/applications/avs/routes.jsx

```diff
@@ -11,8 +12,15 @@ const ErrorBoundaryWrapper = props => (
   </ErrorBoundary>
 );
 
-const routes = [
-  <Route path="/:id" key="/:id" component={ErrorBoundaryWrapper} />,
-];
+const routes = (
+  <Switch>
+    <Route exact path="/:id" key="/:id">
+      <ErrorBoundaryWrapper />
+    </Route>
+    <Route key="404">
```

Comment:
The no-path is [react-router's catch-all/fallback](https://v5.reactrouter.com/web/api/Switch/children-node):

> … A `<Route>` with no path prop or a `<Redirect>` with no from prop will always match the current location.

Reply:
Yes, that is the point. It will be a catch all since it is listed last in the Switch. See Switch documentation.